webpack.config.js

```diff
@@ -1,5 +1,7 @@
 const path = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
+// const ESLintPlugin = require('eslint-webpack-plugin');
+// const StylelintPlugin = require('stylelint-webpack-plugin');
```

Comment:
[ Optional Changes ] Changes marked as optional won't stop your project from being approved but will definitely help improve your codebase if taken into consideration.

You did well in setting up your project and all requirements were met. For that good work, I'm marking this change optional but kindly take it into consideration. 👍 

- [ ] Kindly remove commented codes from your codebase. It's always good to have a clean code with the only meaningful comment that documents a section or feature of your code. Currently, you are having either a code you initially wanted to use but canceled its usage or a linter disabling code. Kindly remove it